controllers/kubevirtmachine_controller_test.go

```diff
@@ -569,7 +574,6 @@ var _ = Describe("reconcile a kubevirt machine", func() {
 
 			setupClient(kubevirt.DefaultMachineFactory{}, objects)
 
-			infraClusterMock.EXPECT().GenerateInfraClusterClient(kubevirtMachine.Spec.InfraClusterSecretRef, kubevirtMachine.Namespace, machineContext.Context).Return(fakeClient, kubevirtMachine.Namespace, nil)
```

Comment:
this change and similar changes makes it look like there are conditions that we no longer test using `v2`

Reply:
```
missing call(s) to *mock.MockInfraCluster.GenerateInfraClusterClient(is equal to nil (*v1.ObjectReference), is equal to  (string), is equal to context.Background (*context.emptyCtx))
```

Reply:
yup, wow, i removed this line locally using v1 and the tests still passed.